sql/core/src/test/resources/sql-tests/inputs/udf/udf-union.sql

```diff
@@ -0,0 +1,56 @@
+-- This test file was converted from union.sql.
+
+CREATE OR REPLACE TEMPORARY VIEW t1 AS VALUES (1, 'a'), (2, 'b') tbl(c1, c2);
+CREATE OR REPLACE TEMPORARY VIEW t2 AS VALUES (1.0, 1), (2.0, 4) tbl(c1, c2);
+
+-- Simple Union
+SELECT *
+FROM   (SELECT * FROM t1
```

Comment:
Can we insert UDF like `udf(c1), udf(c2)`?

Reply:
done